src/System.Management.Automation/engine/parser/Compiler.cs

```diff
@@ -292,6 +286,10 @@ internal static class CachedReflectionInfo
             typeof(ParserOps).GetMethod(nameof(ParserOps.LikeOperator), staticFlags);
         internal static readonly MethodInfo ParserOps_MatchOperator =
             typeof(ParserOps).GetMethod(nameof(ParserOps.MatchOperator), staticFlags);
+        internal static readonly MethodInfo ParserOps_RangeOperator =
+            typeof(ParserOps).GetMethod(nameof(ParserOps.RangeOperator), staticFlags);
+        internal static readonly MethodInfo ParserOps_RangeEnumerator =
```

Comment:
There is a convention here - the field name is class_member - so this should be ParserOps_GetRangeEnumerator.

Reply:
Fixed.